bcp47/library/Data/BCP47.hs

```diff
@@ -79,6 +79,8 @@ module Data.BCP47
   , enUS
   , enTJP
   , enGBTJP
+  , enTJPUpper
+  , enGBTJPUpper
```

Comment:
This is an interesting wrinkle. What we're saying is that "en-t-jp" and "en-T-jp" should be the same thing once they are constructed to a `BCP47`, which I would describe as `BCP47.enTJP`. Therefore, something like `BCP47.enTJPUpper` doesn't make sense to have, IMO. If I were using this library and wanted to grab an `enTJP`, I would be confused by these `Upper` values and not know which to use for what.

Reply:
These are only for testing. I think Evan initially put these here because they use internal constructors. I can _probably_ move them out